docs/guides/isolated_systems.md

```diff
@@ -0,0 +1,223 @@
+
+    b. The production SQLMesh project files are modified with the updates
+
+    c. Execute the `sqlmesh plan staging` command to apply the updates to the `staging` environment (or another environment of your choosing)
+
+    d. Developers validate the set of code updates in the `staging` environment
+
+3. Deploy to production
+
+    a. Execute the `sqlmesh plan` command to apply the updates to the `prod` environment
+
+![Standard SQLMesh workflow with staging environment](./isolated_systems/isolated-systems_standard-workflow-staging.png)
+
+Because the code updates have run in the `staging` environment, the deployment to the `prod` environment (step 3) will not require additional computation due to SQLMesh's virtual data environments. The deployment only requires updates to views so will be very rapid.
+
+### Workflow with isolated systems
+
+We now describe the SQLMesh project workflow with isolated systems.
+
+The primary difference from the single-system workflows described above is that some steps now occur in the nonproduction system and others in the production system.
+
+In some isolated system implementations, the data in the non-production system is different from that in the production system. For example, columns containing sensitive information may be masked or altered in the non-production system's data. These potential differences increase the likelihood of unintentional errors, so we describe (and recommend) a workflow with staging environments.
+
+In the steps below, we assume that the non-production system is the default gateway. The isolated system workflow with staging environments is:
+
+1. Developers modify project code in the non-production environment
+
+    a. Each developer makes their own copy of the production SQLMesh project files
+
+    b. Developers update code in their copy of the project files
+
+    c. Developers create SQLMesh environments and validate their code updates
+
+2. Update production project code and validate in the non-production system
+
+    a. A set of code updates is selected for deployment to production
+
+    b. The production SQLMesh project files are modified with the updates
+
+    c. Execute the `sqlmesh plan` command to apply the updates to its `prod` environment (or another environment of your choosing)
+
+    d. Developers validate the set of code updates in the non-production system `prod` environment
+
+3. Deploy to production system `staging` environment
+
+    a. Execute the `sqlmesh --gateway production plan staging` command to apply the updates to the production system `staging` environment. Note that your production system's gateway may have a different name and that you may choose any environment name other than `prod`.
+
+    b. Developers validate the set of code updates in the production system `staging` environment
+
+4. Deploy to production
+
+    a. Execute the `sqlmesh  --gateway production plan` command to apply the updates to the production system's `prod` environment
```

Comment:
The classifications of breaking/non-breaking are lost across gateways. This must be called out. Reconciliation requires auto classification which is subpar, but apparently the most common. Or use of the github CICD bot to perform interactive classification in the PR.

Reply:
Added info on this on line 171 below - is that sufficient?

Reply:
Yeah!